javascript/javascript1/week1/1.js

```diff
@@ -0,0 +1,77 @@
+//Age-ify (A future age calculator)
+
+const yearOfBirth = 1990;
+const yearFuture = 2050;
+
+const age = yearFuture - yearOfBirth;
+
+console.log('You will be' +" "+ age + ' ' + 'years old in 2050');
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2010;
+const dogYearFuture = 2027;
+const dogsageinfuture = dogYearFuture - dogYearOfBirth;
+const dogyear = dogsageinfuture * 7;
+
+const shouldShowResultInDogYears = true;
+
+if(shouldShowResultInDogYears) {
+    console.log('Your dog will be ' + dogsageinfuture + ' humens years old in 2027');
+    
+} else if(shouldShowResultInDogYears) {
+    console.log('Your dog will be ' + dogyear + ' dog years old in 2027' );
+}
+
+
+//Housey pricey (A house price estimator)
+
+const volumeInMeters = [8*10*10, 5*11*8];
+const gardenSizeInM2 = [100, 70];
+
+const petersPayingPrice = 2500000;
+const juliasPayingPrice = 1000000;
+
+const calculatedPetersHousePrice = volumeInMeters[0] * 2.5 * 1000 + gardenSizeInM2[0] * 300;
+const calculatedJuliasHousePrice = volumeInMeters[1] * 2.5 * 1000 + gardenSizeInM2[1] * 300;
+
+const underPriceDifferencePeter = calculatedPetersHousePrice - petersPayingPrice;
+const  overPriceDifferencePeter = petersPayingPrice - calculatedPetersHousePrice;
+
+const underPriceDifferenceJulia =  calculatedJuliasHousePrice - juliasPayingPrice;
+const  overPriceDifferenceJulia = juliasPayingPrice - calculatedJuliasHousePrice;
+
+
+// Calculations for Peter 
+
+if(calculatedPetersHousePrice > petersPayingPrice) {
+    console.log('Peter paid ' + underPriceDifferencePeter + ' less then calculated price');
```

Comment:
I would rather calculate as part of the if statement the difference between paid price and real price, than make a new variables for it